boards/st/nucleo_f070rb/nucleo_f070rb.yaml

```diff
@@ -19,6 +19,7 @@ supported:
   - spi
   - watchdog
   - rtc
+  - usbd
```

Comment:
@fabiobaltieri There are a few boards without a USB controller, so you probably only want to add the `usbd` test feature if there is also a `usb_device`.

Reply:
yeah most of them in fact, let me see if I got a better list now...